tests/resolvers/post_query/commentsByPost.spec.js

```diff
@@ -0,0 +1,77 @@
+const database = require('../../../db');
+const shortid = require('shortid');
+const User = require('../../../lib/models/User');
+const Organization = require('../../../lib/models/Organization');
+const Post = require('../../../lib/models/Post');
+const Comment = require('../../../lib/models/Comment')
+const commentsByPost = require('../../../lib/resolvers/post_query/commentsByPost')
+
+let user;
+let org;
+let post;
+let comment;
+
+beforeAll(async () => {
+  require('dotenv').config(); // pull env variables from .env file
+  await database.connect();
+  let generatedEmail = `${shortid.generate().toLowerCase()}@test.com`;
+  user = await User.create({
+    firstName: shortid.generate().toLowerCase(),
+    lastName: shortid.generate().toLowerCase(),
+    email: generatedEmail,
+    userType: 'USER',
+    password: 'password',
+  });
+  if (user) {
+    org = await Organization.create({
+      name: 'abcd org',
+      description: shortid.generate().toLowerCase(),
+      isPublic: true,
+      location: 'location',
+      creator: user._id,
+      admins: [user._id],
+      members: [user._id],
+    });
+    if (org) {
```

Comment:
If org was not created?

Reply:
} else {
    throw new Error('User is required to create a organization');
  }

This part covers this case.